app/views/browse/imagestream.html

```diff
@@ -44,105 +44,14 @@ <h1 class="contains-actions">
         <div class="container-fluid">
           <div class="row" ng-if="imageStream">
             <div class="col-md-12">
-              <div class="resource-details">
-                <dl class="dl-horizontal left">
-                  <dt ng-if-start="imageStream.spec.dockerImageRepository">Follows docker repo:</dt>
-                  <dd ng-if-end>{{imageStream.spec.dockerImageRepository}}</dd>
-                  <dt>Docker pull spec:</dt>
-                  <dd>{{imageStream.status.dockerImageRepository}}</dd>
-                </dl>
-                <annotations annotations="imageStream.metadata.annotations"></annotations>
```

Comment:
for reference, this is the annotations directive we use today that handles the show/hide behavior plus how we format annotations when visible